docs/remove-liquidity-v4-etherscan.md

```diff
@@ -0,0 +1,24 @@
+# Remove liquidity from deprecated Ocean Market v4 pools via Etherscan
+
+## Get your balance of pool share tokens
+
+1. Go to the pool's Etherscan/Polygonscan page. To find it without UI, see through your latest transactions and look for Ocean Pool Token (OPT) transfers. Those transactions always come from the pool contract.
+2. On the pool contract page, go to _Contract_ -> _Read Contract_.
+3. Go to field `20. balanceOf` and insert your ETH address. This will retrieve your pool share token balance.
+4. To convert to wei, go to [eth-converter.com](https://eth-converter.com) and input the retrieved pool shares number in the _Ether_ field
```

Comment:
don't really need this step. you can just click on the value and it redirects you to https://rinkeby.etherscan.io/unitconverter?wei=9017293058506014546
![image](https://user-images.githubusercontent.com/26000280/181557184-3c8054f1-ad9b-4ead-97dd-68bc65fc110d.png)

Reply:
And also, this is already in wei, so don't need to convert.

Reply:
excellent, good find, much better